Projects/Claims/Sources/Views/SubmitClaim/SubmitClaimAudioRecordingScreen.swift

```diff
@@ -0,0 +1,353 @@
+import AVFAudio
```

Comment:
I would try to split this screen up, a lot of code in the same file.

Reply:
Will fix